lib/components/Accordion/UserAccordion.tsx

```diff
@@ -3,7 +3,7 @@ import AnimateHeight from 'react-animate-height';
 import { useState } from 'react';
 import { CaretUp, CaretDown } from '@lib/icons';
 
-const Accordion = ({ children, className = '' }: ChildrenProps) => {
+const UserAccordion = ({ children, className = '' }: ChildrenProps) => {
```

Comment:
Can probably delete this component as it is unused though, otherwise, very sexy code.